data_layout.go

```diff
@@ -0,0 +1,421 @@
+package irutil
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// DataLayout is a structural representation of the datalayout
+// string from https://llvm.org/docs/LangRef.html#data-layout
+type DataLayout struct {
+	isBigEndian               bool                                   // default - True
```

Comment:
For consistency with other parts of the llir API, export the fields of `DataLayout` instead of using getters/setters.